gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -146,6 +153,59 @@ static void waitJobInitialization(
     log.info("Work flow {} initialized", workFlowName);
   }
 
+  /**
+   * Inject in some additional properties
+   * @param jobProps job properties
+   * @param inputTags list of metadata tags
+   * @return
+   */
+  public static List<? extends Tag<?>> addAdditionalMetadataTags(Properties jobProps,
```

Comment:
Does this method do anything different from GobblinHelixJobLauncher.addAdditionalMetadataTags() ?

Reply:
Good catch, it's the same function. I just want to move it to utils to leverage it in Helix job callable. Forgot to remove the old function in GobblinHelixJobLauncher, just address it